test/github.go

```diff
@@ -26,6 +26,10 @@ func main() {
 	r := mux.NewRouter()
 	r.HandleFunc("/ping", func(w http.ResponseWriter, r *http.Request) {})
 
+	r.HandleFunc("/api/v3", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("x-github-enterprise-version", "GitHub AE")
+	})
```

Comment:
Does that new feature works with GHES also?

Reply:
This class is for testing only.
And yeah, it should work with GHES as well: see the logic where this header is used to change the requested scope from `public_repo` to `repo` just for GHAE.